test/Writer/FeedTest.php

```diff
@@ -63,45 +63,54 @@ public function testAddsAuthorUriFromArray()
 
     public function testAddAuthorThrowsExceptionOnInvalidNameFromArray()
     {
+        $this->expectException(
+            Writer\Exception\InvalidArgumentException::class
+        );
+
         $writer = new Writer\Feed;
-        try {
-            $writer->addAuthor(['name' => '']);
-            $this->fail();
-        } catch (Writer\Exception\InvalidArgumentException $e) {
-        }
+        $writer->addAuthor(['name' => '']);
     }
 
     public function testAddAuthorThrowsExceptionOnInvalidEmailFromArray()
     {
+        $this->expectException(
+            Writer\Exception\InvalidArgumentException::class
+        );
+
         $writer = new Writer\Feed;
-        try {
-            $writer->addAuthor(['name' => 'Joe',
-                                     'email' => '']);
-            $this->fail();
-        } catch (Writer\Exception\InvalidArgumentException $e) {
-        }
+        $writer->addAuthor(
+            [
+                'name'  => 'Joe',
+                'email' => '',
+            ]
+        );
```

Comment:
minor: for calls like that I'd prefer:
```php
$something->foo([
    'bar' => 'baz',
    'abc' => 'def',
]);
```

Reply:
I am still waiting for the coding standard documentation. 😉